persistence-jdbc/scaladsl/src/test/resources/application.conf

```diff
@@ -0,0 +1,3 @@
+lagom.serialization.play-json {
+  serialization-registry = "com.lightbend.lagom.scaladsl.persistence.TestEntitySerializerRegistry"
+}
```

Comment:
Doesn't matter here because it is just for the local test, will never be concatenated with other .conf files.